README.md

```diff
@@ -341,19 +380,25 @@ Create the instance config file as `instance_config.json` in the `/deploy` direc
 
 Copy `global_config.py` from the root directory into the `/deploy` directory and update all values as needed.
 
+```bash
+# Navigate to the deploy directory.
+cd deploy
+# Copy the global_config.py file into the deploy directory.
+cp ../global_config.py .
+```
+
 3. Environment File
 
 Create the environment file as `.env` in the `/deploy` directory and copy paste the below, updating all values as needed.
 
 ```properties
+DOCKER_HOST=unix:///var/run/docker.sock
 ZEN_WEB_ENV=<project code>
-ZEN_WEB_REMOTE=ubuntu@localhost
 
 ZEN_WEB_HOST=harmony.yourdomain.com
 ZEN_WEB_EMAIL=harmony@yourdomain.com
 
-POSTGRES_DB_URI="<postgresql://harmony-admin:[PASSWORD]@postgres.yourdomain.com:5432/harmony>"
-GRAPHQL_DATABASE_URL="<postgresql://harmony-admin:[PASSWORD_URL_ENCODED]@postgres.yourdomain.com:5432/harmony>"
+DATABASE_URL="<postgresql://harmony-admin:[PASSWORD]@postgres.yourdomain.com:5432/harmony>"
```

Comment:
should this just be the database url created above? the superuser is not necessarily `harmony-admin`

Reply:
Quite right - it's the user, password and database for the intance - and it could be whatever was specified in create_db_setup_script.